### PR TITLE
feature/MovieDetails

### PR DESCRIPTION
**Pull request by megan-venetianer**
## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Refactoring- no new features

## Detailed Description
Sets up the dynamic route for each individual movie details page so that when a user clicks on a movie it will direct them to a different page that shows information for that specific movie. 
Note: the details page still needs to be styled appropriately.

## Why is this change required? What problem does it solve?
N/A

## Were there any challenges that arose while implementing this feature? If so, how were the addressed?
Figuring out how to dynamically route was a little bit tricky. 

## Related Issues:
N/A

## Files modified:

src/components/App/App.js
src/components/MovieCard/MovieCard.js
src/components/MovieContainer/MovieContainer.js
 src/components/MovieDetails/MovieDetails.js
src/components/MovieDetails/MovieDetails.scss
src/sass/application.scss

### CODE REVIEW

**Comment by TimNguyen21** [comment]
Nice work on the set up of the movie detail page.  The route setup for the movie detail page looks good and it allows the application to become more dynamic of where to look for specific movie details.

File: src/components/MovieCard/MovieCard.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import { Link } from 'react-router-dom';
...
-      <h1>{title}</h1>
-      <h2>Average Rating: {averageRating}</h2>
+      <Link to={`/movies/${id}`}>
+        <img src={backdrop} alt={"image for " + title}/>
+        <div className="movie-info-container">
+        <h1>{title}</h1>
+        <h2>Average Rating: {averageRating}</h2>
+      </div>
+      </Link>
     </article>
```
Comment (hborrelli1) [inline src/components/MovieCard/MovieCard.js]: It looks like you have a stray div tag here on line 12. Pretty minor though. Other than that it looks good, nice job!

**Comment by hborrelli1** [comment]
Looks good, nice work!

**Comment by CLPena** [comment]
Agreed, looks great. I'll merge!

**Pull request merged**